sensor/kubernetes/listener/resources/complianceoperator/dispatchers/complianceoperatorscansettingbindings.go

```diff
@@ -56,5 +86,26 @@ func (c *ScanSettingBindings) ProcessEvent(obj, _ interface{}, action central.Re
 			},
 		},
 	}
+
+	if centralcaps.Has(centralsensor.ComplianceV2Integrations) {
+		events = []*central.SensorEvent{
+			{
+				Id:     id,
+				Action: action,
+				Resource: &central.SensorEvent_ComplianceOperatorScanSettingBindingV2{
+					ComplianceOperatorScanSettingBindingV2: &central.ComplianceOperatorScanSettingBindingV2{
+						Id:              id,
+						Name:            scanSettingBindings.Name,
+						ProfileNames:    getProfileNames(scanSettingBindings.Profiles),
+						Conditions:      getStatusConditions(scanSettingBindings.Status.Conditions),
+						ScanSettingName: scanSettingBindings.SettingsRef.Name,
+						Labels:          scanSettingBindings.Labels,
+						Annotations:     scanSettingBindings.Annotations,
+					},
+				},
+			},
+		}
+	}
```

Comment:
```suggestion
		events = append(events, &central.SensorEvent{
			{
				Id:     id,
				Action: action,
				Resource: &central.SensorEvent_ComplianceOperatorScanSettingBindingV2{
					ComplianceOperatorScanSettingBindingV2: &central.ComplianceOperatorScanSettingBindingV2{
						Id:              id,
						Name:            scanSettingBindings.Name,
						ProfileNames:    getProfileNames(scanSettingBindings.Profiles),
						Conditions:      getStatusConditions(scanSettingBindings.Status.Conditions),
						ScanSettingName: scanSettingBindings.SettingsRef.Name,
						Labels:          scanSettingBindings.Labels,
						Annotations:     scanSettingBindings.Annotations,
					},
				},
		})
	}
```
This needs to be an append so that both versions get sent to central.